### ISSUE TITLE
dotnet format warning: Project references without matching metadata references

### ISSUE THREAD

**Issue by rjmurillo** [rjmurillo/moq.analyzers#447]
When running `dotnet format`, the following warning is produced:

```
Warnings were encountered while loading the workspace. Set the verbosity option to the 'diagnostic' level to log warnings.
```

Diagnostic output shows that this warning is related to two specific project references:
- `Moq.CodeFixes.csproj`
- `Moq.Analyzers.Test.Analyzers.csproj`

These references don't have matching metadata references, which is causing the warning. This appears to be a non-critical warning that doesn't affect the functionality of the code formatter.

The warning is currently being tracked as a known issue to inform maintainers and contributors about this behavior.

Steps to reproduce:
1. Run `dotnet format`
2. Observe the warning message

Impact: None - this is a warning only and doesn't affect the functionality of the code formatter.

**Comment by rjmurillo** [comment]
Closing as wontfix per existing label. This is a known `dotnet format` warning about project references without matching metadata references. It does not affect build correctness or analyzer functionality.
